### ISSUE TITLE
Zeus EMP module does not function correctly (prematurely fires with script errors)

### ISSUE THREAD

**Issue by Guac0** [Crowdedlight/Crows-Electronic-Warfare#36]
Mods: CBA_A3, Zeus Enhanced, TFAR Beta, Crows Electronic Warfare.

When you place the "Fire EMP" Module in Zeus, an EMP is fired immediately. The options dialog remains and you can customize the options before pressing "Ok" to fire an EMP in the intended manner. The EMP that fires when you open the dialog has several errors, see Line 784 in the RPT (817 and 829 are other occurrences of it since I opened the dialog additional times closer to the player object to test scope removal). 

Firing it via the "ok" button the dialog seems to not produce any script errors, although I could've sworn there was an error about TFAR during an EMP last week but maybe that's an issue with my group's main modpack... However, I selected the "Remove scopes and electronics" option and my Nightstalker optic wasn't removed, so that might be bugging out as well.

[logfileCrowsEmpErrorNoMods.zip](https://github.com/Crowdedlight/Crows-Electronic-Warfare/files/7448848/logfileCrowsEmpErrorNoMods.zip)

**Comment by Guac0** [comment]
Here's the error that happened last weekend in our main modpack, but I can't replicate it now, and even if I could then it would be almost impossible to fix given that we run 57 mods and the error doesn't occur with only Electronic Warfare and dependencies + TFAR loaded. Feel free to ignore this error.
```
16:53:31 Error in expression <moveItemsArray = ["MineDetector"];

if (crowsEW_zeus_hasTFAR) then {
_removeItem>
16:53:31   Error position: <crowsEW_zeus_hasTFAR) then {
_removeItem>
16:53:31   Error Undefined variable in expression: crowsew_zeus_hastfar
16:53:31 File z\crowsEW\addons\emp\functions\fnc_unitRemoveItems.sqf..., line 140
```

**Comment by Crowdedlight** [comment]
Thanks for reporting. You are indeed right. I did a mistake and forgot to clean up a file. Apologies. 

Should be fixed now and I have pushed the release. 

The second error I could not reproduce and I think that was one thing that was fixed or at least changed in 1.3.3 so it should work correctly. Otherwise don't hesitate to make a new issue on it with details on how to reproduce :)
